tests/integration/targets/test_fqdn_valid/aliases

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+azp/posix/2
+skip/python2.6  # filters are controller only, and we no longer support Python 2.6 on the controller
```

Comment:
We just got rid of Python 2.6 support ;) (Yes, I know, this exists in more places, we'll clean them up eventually.)
```suggestion
```

Reply:
#7534